### DISCUSSION TITLE
cannot fetch from docker

### DISCUSSION THREAD

**Discussion by flbe78** [preactjs/preact#4593] [Q&A]
I am running preact in a Node Docker, and a rest-api in a phpapache docker. 
- docker run --network vumc-network  -p 80:80 --name phpapache phpapache
- docker run -p 4200:4200 --network vumc-network --name node node
When i log in the node docker I can execute:
```
const http = require('http');

http.get('http://phpapache/nl/menu', (res) => {
  let data = '';
  res.on('data', (chunk) => {
    data += chunk;
  });
  res.on('end', () => {
    console.log(data);
  });
}).on('error', (error) => {
  console.error(error);
});
```
which correctly returns a json.
But in de react-app is a function that does something similar:

```
fetch('http://phpapache/nl/menu')
      .then(res => res.json())
      .catch(error => {
        console.error(
          "There was a problem loading the database from DatabasesNav",
          error
        );
      });
```
which fails with an error: `TypeError: Failed to fetch`

**Comment by JoviDeCroock** [comment]
This has nothing to do with Preact 😅 This might be due to a CORS/networking error but we don't have any impact on how your fetches work.
